### DISCUSSION TITLE
Would an update to .net 7 and 8 be coming soon?

### DISCUSSION THREAD

**Discussion by acrigney** [IronLanguages/ironpython3#1763] [Q&A]
Thanks heaps for all your hard work guys!
Just wondering if you are planning any upgrades to .net 7/8 and any pathway for future releases?

**Comment by slozier** [comment]
Once I find a bit of time I'll probably be adding a .NET 8 target, but there's no reason you couldn't use the .NET 6 assemblies with .NET 7/8... Unless there are some issues blocking you from doing this?

**Comment by acrigney** [comment]
Awesome mate I didn't know that they would be compatible, I would have
thought that you would have to use .net standard for that.
